### PR TITLE
Allow modification of tablet unhealthy_threshold via debugEnv

### PR DESCRIPTION
**Pull request by aquarapid**
WDYT?  Obviously not very clean, so open to suggestions.  One thing is that modifying the Config flag isn't atomic, but that's probably fine, since we don't read from it after Init?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -93,6 +93,7 @@ require (
 	github.com/uber/jaeger-client-go v2.16.0+incompatible
 	github.com/uber/jaeger-lib v2.0.0+incompatible // indirect
 	github.com/z-division/go-zookeeper v0.0.0-20190128072838-6d7457066b9b
+	go.uber.org/atomic v1.4.0
```
Comment (sougou) [inline go.mod:96]: recommend using sync2 to avoid an additional dependency.

**Review by sougou — COMMENTED** [review]
Looks good. One nit.

**Review by sougou — APPROVED** [review]

**Comment by deepthi** [comment]
@aquarapid does this need to go into 10.0?

**Comment by aquarapid** [comment]
@deepthi it's a bit late, but yes, if possible.

**Comment by deepthi** [comment]
Oops! Forgot that we need an issue. Can you create one?

**Pull request merged**